pkg/util/clusteragent/commands/clusterchecks.go

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build clusterchecks
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/flare"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+func GetClusterChecksCobraCmd(flagNoColor *bool, confPath *string, loggerName config.LoggerName) *cobra.Command {
+	clusterChecksCmd := &cobra.Command{
+		Use:   "clusterchecks",
+		Short: "Prints the active cluster check configurations",
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			if *flagNoColor {
+				color.NoColor = true
+			}
+
+			// we'll search for a config file named `datadog-cluster.yaml`
+			config.Datadog.SetConfigName("datadog-cluster")
+			err := common.SetupConfig(*confPath)
+			if err != nil {
+				return fmt.Errorf("unable to set up global cluster agent configuration: %v", err)
```

Comment:
First letter uppercase

Reply:
This was actually lowercase initially, I just moved the code... I guess it's better to leave it lowercase (although I do agree that the inconsistency is not great)